changelog_unreleased/handlebars/pr-7051.md

```diff
@@ -1,4 +1,4 @@
-#### handlebars: Fix superfluous line breaks in ConcatStatement ([7051](https://github.com/prettier/prettier/pull/7051) by [@dcyriller](https://github.com/dcyriller))
+#### handlebars: Fix superfluous line breaks in ConcatStatement ([#7051](https://github.com/prettier/prettier/pull/7051) by [@dcyriller](https://github.com/dcyriller))
```

Comment:
Can you please also validate that headers 1) have four `#`, 2) don't start with redundant prefixes like "handlebars:", "HTML:", etc.?

Reply:
Done.

Reply:
yes

Reply:
Done https://github.com/prettier/prettier/pull/7320/commits/baa548761a13638de335d1741a17fe601cf96626